CHANGELOG.md

```diff
@@ -1,5 +1,75 @@
 # go-filecoin changelog
 
+## go-filecoin 0.5.4
+
+We're happy to announce go-filecoin 0.5.4. Highlights include an updated Proof-of-Spacetime implementation and an upgrade-capable network.
+
+### Features
+
+#### 🌳 Network upgrade capability
+
+Two changes have been made to enable software releases without restarting the network. First, a network name is now embedded in the genesis state, permitting multiple networks to follow different upgrade schedules. In addition, the Git SHA compatibility check has been removed from the Hello protocol, enabling nodes with different, but compatible, code to interoperate. Going forward, the user devnet will no longer be restarted with every software release; it will still be restarted as-needed.
+
+#### 🚀 Updated Proof-of-Spacetime (PoSt)
+
+A new proof construction, [Rational PoSt](https://github.com/filecoin-project/specs/blob/master/proof-of-spacetime.md), has been [implemented](https://github.com/filecoin-project/rust-fil-proofs/pull/763) and [integrated](https://github.com/filecoin-project/go-filecoin/pull/3318). This construction is the same shape as our candidate for testnet and resolves outstanding limitations on proving over many sectors.
+
+#### 🎟️ Block and consensus changes
+
+Block headers are now signed by miners, and election tickets form an array in each header.
+
+#### 🔗 Chain status command
+
+`go-filecoin chain status` is a new command that provides insight into chain sync progress.
+
+### Performance and Reliability
+
+#### ⚡Chain validation performance
+
+Previously in go-filecoin 0.4, we aimed to speed up chain syncing by focusing on the first phase: chain fetching. go-filecoin 0.5 continues that work with improvements to the second phase: chain validation. By switching from HAMT bitwidth 8 to HAMT bitwidth 5, we see a general average improvement in benchmarks of about 4-to-1, across memory usage, speed of operations, and bytes written to disk. Users are encouraged to measure and share their own benchmarks.
```

Comment:
If we want to mention to cbor-gen, we could add a sentences that says something like:
"In addition, optimizations to encoding and decoding of HAMT data structures may result in additional performance improvements"

Reply:
added, thanks